gitstrap get teams crashes on nil pointer

When running `gitstrap get teams --org artipie` command (I have access to artipie org), it crashes with log:
```
gitstrap get teams --org artipie
panic: assignment to entry in nil map

goroutine 52 [running]:
github.com/g4s8/gitstrap/internal/gitstrap.(*Gitstrap).GetTeams.func1()
	/home/g4s8/go/pkg/mod/github.com/g4s8/gitstrap@v0.0.0-20210830100349-249655e5f45f/internal/gitstrap/get.go:152 +0x4a5
created by github.com/g4s8/gitstrap/internal/gitstrap.(*Gitstrap).GetTeams
	/home/g4s8/go/pkg/mod/github.com/g4s8/gitstrap@v0.0.0-20210830100349-249655e5f45f/internal/gitstrap/get.go:125 +0x15b
```

--- a/internal/spec/model.go
+++ b/internal/spec/model.go
@@ -63,11 +63,9 @@ func NewModel(kind Kind) (*Model, error) {
 	if err := kind.validate(); err != nil {
 		return nil, err
 	}
-	m := new(Model)
-	m.Version = Version
-	m.Kind = kind
-	m.Metadata = new(Metadata)
-	return m, nil
+	meta := new(Metadata)
+	meta.Annotations = make(map[string]string)
+	return &Model{Version: Version, Kind: kind, Metadata: meta}, nil
 }
 
 type errUnknownKind struct {

